index.js

```diff
@@ -1,21 +1,37 @@
 /* @flow */
-import fs from 'react-native-fs';
+import fs from "react-native-fs";
+import { Platform } from "react-native";
 
 export const DocumentDir = fs.DocumentDirectoryPath;
 export const CacheDir = fs.CachesDirectoryPath;
+const OSPathSeparator = Platform.select({
+  ios: "/",
+  android: "/",
+  windows: "\\",
+});
```

Comment:
is it possible to use `path`? if so you can just use `path.sep` if you need access to the separator

Reply:
I don't think so as it's React-Native JS world and not node.js?